types/qlik-engineapi/index.d.ts

```diff
@@ -6915,6 +7333,27 @@ declare namespace EngineAPI {
         getUniqueID(): Promise<string>;
 
         /**
+         * Import an App
+         * @param qAppId - new AppId
+         * @param qSrcPath - source path
+         * @param qIds - ???
+         * Note: from shema file
+         * Note: This operation is possible only in Qlik Sense Enterprise.
+         */
+        importApp(qAppId: string, qSrcPath: string, qIds: string[]): Promise<void>;
+
+        /**
+         * Import an App Extended
+         * @param qAppId - new AppId
+         * @param qSrcPath - source path
+         * @param qIds - ???
```

Comment:
Could just omit the documentation if you don't know. Same for `no / empty docu` comments below.

Reply:
@andy-ms yea, but I can not search for not existing params, it is easier to search for a mark and than solve it. If some things are not documented we try to find out the point and fix it step by step